internal/auth/repository/eventsourcing/eventstore/auth_request.go

```diff
@@ -482,9 +485,16 @@ func (repo *AuthRequestRepo) fillPolicies(ctx context.Context, request *domain.A
 	if orgID == "" {
 		orgID = request.UserOrgID
 	}
+	privateLabelingOrgID := orgID
+	if privateLabelingOrgID == "" {
+		privateLabelingOrgID = request.RequestedPrivateLabelingOrgID
+	}
```

Comment:
is this order really intentioned for private labeling?
1. primary domain scope
2. org from user
3. private label scope
4. iam

Reply:
yes, do you think it should be in another order?